core/src/main/java/org/hibernate/ogm/type/impl/ByteStringType.java

```diff
@@ -28,7 +28,7 @@ public ByteStringType() {
 
 	@Override
 	public String getName() {
-		return "byte_integer";
+		return "byte_as_string";
 	}
```

Comment:
They are in separate commits, Isn't that enough?

Also, this is the only change that it's not required for OGM-366, but it is so small that I decided not to open a separate JIRA for it. I guess I will send a separate pull request or remove the commit